docs/app/Navbar.tsx

```
@@ -0,0 +1,385 @@
+                        <button
+                            id={"themeSwitcher"}
+                            className={"btn btn-yscope-secondary btn-sm d-inline-flex " +
+                                "align-items-center"}
+                            style={{
+                                border: "none",
+                                justifyContent: "center",
+                                padding: "0.25rem 0.5rem",
+                                minWidth: "28px",
+                            }}
+                            onClick={toggleTheme}
+                        >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)